imap_processing/ultra/l1b/lookup_utils.py

```diff
@@ -0,0 +1,158 @@
+"""Contains tools for lookup tables for l1b."""
+
+import numpy as np
+import pandas as pd
+
+from imap_processing import imap_module_directory
+
+base_path = f"{imap_module_directory}/ultra/lookup_tables"
```

Comment:
Keeping paths as `Path` objects is platform agnostic.
```suggestion
base_path = imap_module_directory / "ultra" / "lookup_tables"
```